src/algorithms/postprocessing/eqOdds.jl

```diff
@@ -17,6 +17,12 @@ function EqOddsWrapper(classifier::MLJBase.Model; grp::Symbol=:class)
 	return EqOddsWrapper(grp, classifier)
```

Comment:
Finicky point: According to the MLJ API *every* hyper-parameter (including `classifier`) should be a key-word argument. That is, the user does `EqOddsWrapper(classifier=..., grp=...)`. If `classifier` is not specified by user (default=`nothing`) have clean! throw an error "You must specify model=..." . 

Also  the keyword constructor needs to call clean!, as shown [here](https://alan-turing-
institute.github.io/MLJ.jl/dev/adding_models_for_general_use/#New-model-type-declarations-and-optional-clean!-method-1) (but `fit` does not need to do this).

Reply:
I saw that MLJ internally calls the clean! function. So I don't see why we need to explicitly call it.

I will fix the API pattern for keyword arguments in a separate PR.

Reply:
MLJ fit! calls clean! but the user ought to know at construction time that his parameters are no good, right?